hadoop-hdds/rocksdb-checkpoint-differ/src/test/java/org/apache/ozone/rocksdb/util/TestManagedSstFileReader.java

```diff
@@ -65,7 +65,7 @@ class TestManagedSstFileReader {
   private String createRandomSSTFile(TreeMap<String, Integer> keys)
       throws IOException, RocksDBException {
     File file = File.createTempFile("tmp_sst_file", ".sst");
-    file.deleteOnExit();
+//    file.deleteOnExit();
```

Comment:
Is it intentional? If not, please revert it.

Reply:
done